.gitignore

```diff
@@ -2,3 +2,6 @@
 /build/
 node_modules
 npm-debug.log
+
+# Intellij
+.idea/
```

Comment:
This shouldn't be in the project .gitignore, but your personal .gitignore. Otherwise this file would be full of ignores for every editor that contributors use. Please remove the changes in this file.